File: SWEA-python/[2072] 홀수만 더하기/Solution.py
T = int(input())
result = []

for test_case in range(1, T+1):
    num = list(map(int, input().split()))
    sum = 0
    for j in range(len(num)):
        if num[j-1]%2==1:
            sum += num[j-1]
    result.append(sum)

for i in range(1, T+1):        
    print(f"#{i} {result[i-1]}")

File: SWEA-python/[2072] 홀수만 더하기/Solution.c
#include <stdio.h>

int main(void) {
    int T;
    scanf("%d", &T);

    for (int test_case = 1; test_case <= T; test_case++) {
        int sum = 0;
        int num;

        for (int i = 0; i < 10; i++) {
            scanf("%d", &num);
            if (num % 2 != 0) { 
                sum += num;
            }
        }

        printf("#%d %d\n", test_case, sum);
    }

    return 0;
}